hauberk.app.domain-verification.json

```diff
@@ -0,0 +1,53 @@
+{
+    "providerId": "hauberk.app",
+    "providerName": "Hauberk",
+    "serviceId": "domain-verification",
+    "serviceName": "Hauberk Domain Verification",
+    "version": 1,
+    "logoUrl": "https://assets.hauberk.app/hauberkLogo.svg",
+    "syncPubKeyDomain": "hauberk.app",
+    "description": "Configures DNS records for Hauberk domain verification",
+    "variableDescription": "unique is randomly generated for use in the username part of the dmarc rua email address; publicKey is a cryptographic signature for signing DKIM; region is an AWS region.",
+    "syncBlock": false,
+    "syncRedirectDomain": "hauberk.app",
+    "records": [
+        {
+            "groupId": "outbound",
+            "type": "MX",
+            "host": "send.%fqdn%",
+            "pointsTo": "feedback-smtp.%region%.amazonses.com",
+            "priority": 10,
+            "ttl": "auto"
```

Comment:
ttl must be a number.
@kerolasa would you check why the linter didn't detect this error?